packages/tree-sitter/build.sh

```diff
@@ -21,45 +21,27 @@ termux_pkg_auto_update() {
 		return
 	fi
 
-	# Figure out the new SHA256 for the `tree-sitter-cross-tools` release file.
-	local TS_GZ_URL TS_GZFILE NEW_TS_SHA256
-	TS_GZ_URL="https://github.com/tree-sitter/tree-sitter/releases/download/v${latest_release}/tree-sitter-linux-x64.gz"
-	TS_GZFILE="$(mktemp)"
-	curl -sL "$TS_GZ_URL" -o "$TS_GZFILE"
-	NEW_TS_SHA256=$(sha256sum "${TS_GZFILE}" | cut -d' ' -f1)
-
-	local TS_GZ_SHA256=c300ea9f2ca368186ce1308793aaad650c3f6db78225257cbb5be961aeff4038
-
-	if [[ "$NEW_TS_SHA256" != "$TS_GZ_SHA256" ]]; then
-		echo "Need to update SHA256 sum for tree-sitter-host-tools"
-		echo "-$TS_GZ_SHA256"
-		echo "+$NEW_TS_SHA256"
-		sed \
-			-e "s|local TS_GZ_SHA256=${TS_GZ_SHA256}|local TS_GZ_SHA256=${NEW_TS_SHA256}|" \
-			-i "${TERMUX_SCRIPTDIR}/packages/tree-sitter/build.sh"
-	fi
-
-	termux_pkg_upgrade_version "${latest_release}"
-}
```

Comment:
Okay, I see what happened now to cause the error in https://github.com/termux/termux-packages/commit/2b19132a000694db6133d8aa78c68d9e9865a66b.

When I migrated this hunk to `termux_setup_treesitter()` I did a refactor of the variable names, caught it in the first part of the `sed` substitution, missed it in the second.
So when that codepath got hit for the first time yesterday that missed rename caused it to rename the variable, which would have (silently, until something tries to use `termux_setup_treesitter`) broken the setup auto-update for future versions.